.github/workflows/dotnet.yml

```
@@ -14,8 +14,14 @@ on:
       - 'lib/mac/**'
       - 'lib/raspberry-pi/**'
       - 'lib/windows/**'
-      - 'resources/**'
-      - '!resources/lint/**'
+      - 'resources/audio_samples/**'
+      - 'resources/keyword_files/beaglebone/**'
+      - 'resources/keyword_files/jetson/**'
+      - 'resources/keyword_files/linux/**'
+      - 'resources/keyword_files/mac/**'
+      - 'resources/keyword_files/raspberry-pi/**'
+      - 'resources/keyword_files/windows/**'
```

I don't think this does what we want it to do since it only covers the `en` case right? We should switch to something like this (assuming actions supports the syntax) for all the workflows:
```suggestion
      - 'resources/keyword_files*/beaglebone/**'
      - 'resources/keyword_files*/jetson/**'
      - 'resources/keyword_files*/linux/**'
      - 'resources/keyword_files*/mac/**'
      - 'resources/keyword_files*/raspberry-pi/**'
      - 'resources/keyword_files*/windows/**'
```

---

good point!